nomad/vault.go

```diff
@@ -474,13 +477,10 @@ func (v *vaultClient) renewalLoop() {
 		case <-authRenewTimer.C:
 			// Renew the token and determine the new expiration
 			err := v.renew()
-			currentExpiration := v.lastRenewed.Add(time.Duration(v.tokenData.CreationTTL) * time.Second)
+			currentExpiration := v.tokenData.ExpireTime
```

Comment:
This is never getting updated; renewal will break

Reply:
Good catch - thanks.  I think we'll need to update the field to protect against leases TTLs being different from creation ttl and/or when we get closer to the token ttl.  I'll rework this.